core/src/test/scala/org/apache/spark/serializer/KryoSerializerSuite.scala

```diff
@@ -399,6 +399,14 @@ class KryoSerializerSuite extends SparkFunSuite with SharedSparkContext {
     assert(!ser2.getAutoReset)
   }
 
+  private def testBothUnsafeAndSafe(f: SparkConf => Unit): Unit = {
```

Comment:
At first glance I don't see this function used anywhere I'm assuming this was meant to wrap some of the tests?